fl4health/clients/evaluate_client.py

```diff
@@ -98,7 +107,10 @@ def evaluate(self, parameters: NDArrays, config: Config) -> Tuple[float, int, Di
             metric_values,
         )
 
-    def _handle_logging(self, losses: Losses, metrics_dict: Dict[str, Scalar], is_global: bool) -> None:
+    def _handle_logging(  # type: ignore
```

Comment:
Is this type ignore to handle the fact that the signatures don't match here?

Reply:
Yeah the signatures don't match so it won't pass unless we type ignore